### DISCUSSION TITLE
ram-rom-sdcard-internal-memory/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#1] [Announcements]
# https://aprd.ir/ram-rom-sdcard-internal-memory/

تفاوت رم با رام با کارت حافظه و حافظه داخلی چیه؟

https://aprd.ir/ram-rom-sdcard-internal-memory/

**Comment by mrhp1975** [comment]
سلام امیرجان.
 من قصد ادامه تحصیل در مقطع ارشد را داشتم،اما چندی قبل یک کلیپ از دکتر شعبانعلی درباره ادامه تحصیل در مقطع ارشد و دکترا دیدم و تا حدی مردد و منصرف شدم. دوست داشتم اگر ممکن هست نظر شما و تجربه شما را بدانم درباره ی ادامه تحصیل در مقطع ارشد.

**Reply by pourmand1376** [reply]
سلام و درود بر شما.
انصافا چه سوال سختی پرسیدی. من آقای شعبانعلی رو استاد خودم می‌دونم و خیلی ازشون چیز یاد گرفتم و می‌گیرم. علاوه بر اون کلیپی که می‌فرمایید [اینجا ](https://mrshabanali.com/%D8%AF%D8%B1%D8%A8%D8%A7%D8%B1%D9%87-%D9%85%D9%87%D8%A7%D8%AC%D8%B1%D8%AA-%D9%88-%D8%A7%D8%AF%D8%A7%D9%85%D9%87-%D8%AA%D8%AD%D8%B5%DB%8C%D9%84/)و [اینجا ](https://mrshabanali.com/%da%86%d8%b1%d8%a7-%d8%af%da%a9%d8%aa%d8%b1%d8%a7-%d9%86%d9%85%db%8c%d8%ae%d9%88%d8%a7%d9%86%db%8c%d8%9f/) و [اینجا ](https://mrshabanali.com/%D8%A7%D8%B3%D8%AA%D8%B1%D8%A7%D8%AA%DA%98%DB%8C-%D8%A7%D8%AF%D8%A7%D9%85%D9%87-%D8%AA%D8%AD%D8%B5%DB%8C%D9%84/)هم گریزی به این مسئله زدند.کامنت‌های وبلاگ‌شون هم خیلی مفیده.

من نمی‌تونم به شما بگم که ارشد یا دکترا بخونی یا نه. تنها کاری که می‌تونم بکنم اینه که توضیح بدم من چرا تصمیم گرفتم ارشد و دکترا بخونم. شاید چیزهایی که میگم کمک کننده باشه. به زودی در این رابطه مطلبی می‌نویسم. علی الحساب می‌تونم بگم که «سربازی» و «تصمیم به مهاجرت» از مهم‌ترین عواملش هست.

**Comment by mrhp1975** [comment]
ممنون امیر جان از پاسخت،
خیلی دوست دارم و منتظرم مطلبی بنویسی درباره دانشگاه و تحصیل در مقطع ارشد و
دکتری چون قبلا هم دیده بودم جایی نوشته بودی " کورس دانشگاهی پاس کردن خیلی
کار عبثی هست. عبث به معنای واقعی کلمه. خیلی وقته کورس خوب نگذروندم. حتی شریف
هم کورس خوب نداشت به نظرم"
در تاریخ دوشنبه ۶ فوریهٔ ۲۰۲۳،‏ ۱۳:۵۴ Amir Pourmand <
***@***.***> نوشت:
